conf/base.config

```diff
@@ -23,12 +23,26 @@ process {
     errorStrategy = { task.exitStatus in [143,137] ? 'retry' : 'ignore' }
   }
 
+  //"make_classifier" is the older version (just one step) of preparing the database for classification
   withName: make_classifier {
     cpus = { check_max (1 * task.attempt, 'cpus' ) }
     memory = { check_max (42.GB * task.attempt, 'memory' ) }
     time = { check_max (12.h * task.attempt, 'time' ) }
   }
 
+  //"clfr_extract_seq" and "clfr_train" is the newer version, where database preparation is split in two
```

Comment:
As I see it, you deleted `process make_classifier`, so this can be removed here as well.
```suggestion
```